packages/shared/components/modals/ProfileActions.svelte

```diff
@@ -61,31 +58,14 @@
 
     const syncLedgerDeviceStatus = (): void => {
         isCheckingLedger = true
-        // const _onComplete = () => setTimeout(() => (isCheckingLedger = false), 500)
-        // getLedgerDeviceStatus(false, _onComplete, _onComplete, _onComplete)
+        const _onComplete = () => setTimeout(() => (isCheckingLedger = false), 500)
+        getLedgerDeviceStatus(_onComplete, _onComplete, _onComplete)
     }
 
     const updateLedgerConnectionText = (): void => {
-        const values: LocaleArguments =
-            $ledgerDeviceState === LedgerDeviceState.LegacyConnected ? { legacy: LedgerAppName.IOTALegacy } : {}
-        const text = localize(`views.dashboard.profileModal.hardware.statuses.${$ledgerDeviceState}`, { values })
+        const text = localize(`views.dashboard.profileModal.hardware.statuses.${$ledgerDeviceStatus.connectionState}`)
 
-        /**
-         * NOTE: The text for when another app (besides IOTA or IOTA Legacy) is open
-         * requires an app name to be prepended or else the text won't make sense.
-         */
-        if ($ledgerDeviceState === LedgerDeviceState.OtherConnected) {
-            // getLedgerOpenedApp()
-            //     .then((la: LedgerApp) => {
-            //         ledgerConnectionText = `${la.name} ${text}`
-            //     })
-            //     .catch((err) => {
-            //         ledgerDeviceState.set(LedgerDeviceState.NotDetected)
-            //         console.error(err)
-            //     })
-        } else {
-            ledgerConnectionText = text
-        }
+        ledgerConnectionText = text
```

Comment:
We can leave the `text` variable declaration if we need to use it later, but otherwise could just set it directly with `ledgerConnectionText = localize(`views.dashboard.profileModal.hardware.statuses.${$ledgerDeviceStatus.connectionState}`)`